etl_pipeline/outages/load_outages.py

```diff
@@ -0,0 +1,216 @@
+"""
+Load outage data into RDS
+
+- Reads DB creds from .env (DB_HOST, DB_PORT, DB_DB, DB_USER, DB_PASSWORD)
+- Upserts `outage`
+- Inserts `postcode` (ignore duplicates)
+- Inserts `outage_postcode_link` by staging (outage_id, postcode) into a TEMP table
+  and resolving postcode_id
+"""
+
+from __future__ import annotations
+
+import json
+import os
+from typing import Any, Dict, Iterable, List, Tuple
+
+import pandas as pd
+import psycopg2
+from psycopg2.extras import execute_values
+from dotenv import load_dotenv, find_dotenv
+from transform_outages import transform_outages
+from extract_outages_csv import generate_outage_csv
+
+
+def load_db_config() -> Dict[str, Any]:
```

Comment:
Would it not be easier just making this a connection function? 
```suggestion
def get_db_connection():
    """Connect to the postgres database managed by RDS."""
    load_dotenv()

    return connect(database=ENV["DB_NAME"],
                   user=ENV["DB_USERNAME"],
                   password=ENV["DB_PASSWORD"],
                   host=ENV["DB_HOST"],
                   port=ENV["DB_PORT"])
```